quickwit/quickwit-cluster/src/cluster.rs

```diff
@@ -269,15 +269,28 @@ impl Cluster {
             .set(key, value);
     }
 
+    /// Sets a key-value pair on the cluster node's state.
+    pub async fn set_self_key_value_delete_after_ttl(
+        &self,
+        key: impl Display,
+        value: impl Display,
```

Comment:
```suggestion
        key: impl ToString,
        value: impl ToString,
```
nit: `Display` implements `ToString`, but `ToString` is more generic.

Reply:
done